orbitize/priors.py

```diff
@@ -101,7 +101,7 @@ def compute_lnprob(self, element_array):
 
         if self.no_negatives:
 
-            bad_samples = np.where(samples <= 0)[0]
+            bad_samples = np.where(element_array <= 0)[0]
```

Comment:
I know that there are issues with checking for equality when you have floating point numbers, so this probably makes no difference, but shouldn't the check here be for `element_array < 0` instead of `element_array <= 0`, just so it reads more clearly? Unless we do want to exclude zero specifically?

Reply:
Sure, that makes sense. I'll push that change to this branch. Thanks!